src/flags.ts

```diff
@@ -24,8 +25,17 @@ export const perflogFlag = Flags.boolean({
   },
 });
 
+// Modifying a custom flag is an anti-pattern. Creating a new
+// custom flag is the preferred approach. But if it must be done,
+// then the type must be asserted so that @oclif/core can properly
+// parse the flag's type.
+const targetOrg = {
+  ...requiredOrgFlagWithDeprecations,
+  summary: messages.getMessage('flags.targetOrg.summary'),
+} as Interfaces.OptionFlag<Org>;
+
 export const orgFlags = {
-  'target-org': { ...requiredOrgFlagWithDeprecations, summary: messages.getMessage('flags.targetOrg.summary') },
```

Comment:
I don't think the summary in this plugin `Org alias or username to use for the target org.` needs to exist.

https://github.com/salesforcecli/plugin-data/blob/main/messages/messages.md#flagstargetorgsummary

since the default is `Username or alias of the target org.`
https://github.com/salesforcecli/sf-plugins-core/blob/main/messages/messages.md#flagstargetorgsummary